ReactiveXComponentTest/UnitTests/ParserTests/XCApiConfigParserTest.cs

```diff
@@ -114,11 +114,23 @@ public void GetConsumerTopic_GiventComponentAndStateMachien_ShouldReturnAConsume
 
             const string stateMachine = "HelloResponse";
             const string expectedTopic = "output.1_0.HelloMicroservice.HelloWorld.HelloResponse";
-            var topic = _xcApiConfigParser.GetConsumerTopic(_component, stateMachine);
+            var topic = _xcApiConfigParser.GetSubscriberTopic(_component, stateMachine);
 
             Check.That(topic).IsEqualTo(expectedTopic);
         }
 
+        [Test]
+        public void GetSerializationType_ShouldReturnSerializationType_Test()
+        {
+            _xcApiConfigParser = new XCApiConfigParser();
+            _xcApiConfigParser.Parse(_xcApiStream);
+
+            var expectedSerialization = "Binary";
+            var serialization = _xcApiConfigParser.GetSerializationType();
+
+            Check.That(serialization).IsEqualTo(expectedSerialization);
+        }
+
         private void Dispose(bool disposing)
```

Comment:
No need to implement IDisposable for a unit test. Just rename the Dispose() method and tag it with the [TearDown] attribute.